studentInfo.py

```diff
@@ -0,0 +1,72 @@
+  def __init__(self):
+    self.assignRandomK()
+    self.assignRandomD()
+    self.assignRandomA()
+    self.randomChampion()
+    self.randomLane()
+    self.randomItem1()
+    self.randomItem2()
+    self.randomItem3()
+    self.randomItem4()
+    self.randomItem5()
+    self.randomItem6()
+
+  def randomLane(self):
+     self.lane = random.choice(self.lanes)
+
+  def randomChampion(self):
+     self.mainChamp = random.choice(self.mainChamps)
+  
+  def assignRandomK(self):
+     self.K = random.randint(0,30)
+  def assignRandomD(self):
+     self.D = random.randint(0,30)
+  def assignRandomA(self):
+     self.A = random.randint(0,30)  
+  def randomItem1(self):
+    self.item1 = random.choice(self.fullItems)
+  def randomItem2(self):
+    self.item2 = random.choice(self.fullItems)  
+  def randomItem3(self):
+    self.item3 = random.choice(self.fullItems)
+  def randomItem4(self):
+    self.item4 = random.choice(self.fullItems)
+  def randomItem5(self):
+    self.item5 = random.choice(self.fullItems)
+  def randomItem6(self):
+    self.item6 = random.choice(self.fullItems)
+
+inputQuestions = [ 
+  "For CHAMPIONS, type 0",
+]
+
+def getUserSelection():
+  print (inputQuestions[0])
+  return int(input("Type selection and press enter: "))
+
+def RandomizedChampions():
+  count = int( input("How Many Randomized Champions? "))
+  for i in range(0, count):
+    champ = Champion()
+    print (str(champ.mainChamp) + " " + str(champ.lane) + ", " + str(champ.item1) + ", " + str(champ.item2) + ", " + str(champ.item3) + ", " + str(champ.item4) +", " + str(champ.item5) + ", " + str(champ.item6))
+def printHeader():
+    print("League Champion Build Generator")
+main()
```

Comment:
[[PEP 8]](https://www.python.org/dev/peps/pep-0008/)

It's recommended to have a line break at the end of a file.